src/sherpa_ai/memory/belief.py

```diff
@@ -55,6 +55,7 @@ def get_context(self, token_counter: Callable[[str], int], max_tokens=4000):
             if (
                 event.event_type == EventType.task
                 or event.event_type == EventType.result
+                or event.event_type == EventType.user_input
```

Comment:
more concise: if  event.event_type in ( EventType.task,  EventType.task, EventType.user_input)